src/ParseObject.js

```diff
@@ -34,10 +33,12 @@ import {
 import ParsePromise from './ParsePromise';
 import ParseQuery from './ParseQuery';
 import ParseRelation from './ParseRelation';
+import * as SingleInstanceState from './SingleInstanceState';
```

Comment:
nah, makes sense, though with every extra word I worry about things getting too Java-y. I'll go find-replace.

AbstractInstanceStateControllerFactory, here we come!